src/main/java/com/softserve/academy/event/repository/impl/QuestionRepositoryImpl.java

```diff
@@ -3,7 +3,19 @@
 import com.softserve.academy.event.entity.SurveyQuestion;
 import com.softserve.academy.event.repository.QuestionRepository;
 import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
 
 @Repository
-public class QuestionRepositoryImpl extends BasicRepositoryImpl<SurveyQuestion, Long> implements QuestionRepository {
+@Transactional
```

Comment:
Transactional set's to service, not repository

Reply:
fixed